projects/plugins/jetpack/modules/custom-css/custom-css-4.7.php

```diff
@@ -554,11 +581,11 @@ public static function sanitize_css( $css, $args = array() ) {
 				$css = wp_kses_split( $css, array(), array() );
 				$css = str_replace( '&gt;', '>', $css ); // kses replaces lone '>' with &gt;
 				// Why both KSES and strip_tags?  Because we just added some '>'.
-				$css = strip_tags( $css );
+				$css = strip_tags( $css ); // phpcs:ignore WordPress.WP.AlternativeFunctions.strip_tags_strip_tags -- scared to update this to wp_strip_all_tags since we're building a CSS file here.
 
-				if ( $css != $prev ) {
-					$warnings[] = 'kses found stuff';
-				}
+			if ( $css !== $prev ) {
+				$warnings[] = 'kses found stuff';
+			}
```

Comment:
Same as above, the indenting was technically correct before but the real fix is to remove the extraneous braces and unindent the whole block.

Reply:
Forget this one?

Reply:
Hmm I though I fixed this one - did I just forget to mark the conversation as resolved?

Reply:
I didn't see the change earlier. It's possible I screwed up getting GH's UI to show the diff of just what changed.